products/models.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models
+
+class Product(models.Model):
+    name            =   models.CharField(max_length=45)
+    subcategory     =   models.ForeignKey('SubCategory', on_delete = models.SET_NULL, null=True)
+    stock           =   models.PositiveIntegerField()
+    price           =   models.DecimalField(max_digits=10, decimal_places=2)
+    discount_rate   =   models.OneToOneField('DiscountRate', on_delete = models.SET_NULL, null=True)
+    category        =   models.ForeignKey('Category', on_delete = models.SET_NULL, null=True)
+    created_at      =   models.DateTimeField(auto_now_add=True)
+    updated_at      =   models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table    = 'products'
+
+class Category(models.Model):
+    name            =   models.CharField(max_length=45)
+
+    class Meta:
+        db_table    = 'categories'
+
+class SubCategory(models.Model):
+    name            = models.CharField(max_length=45)
+    category        = models.ForeignKey('Category', on_delete = models.SET_NULL, null=True)
+
+    class Meta:
+        db_table    = 'subcategories'
+
+class Allergy(models.Model):
+    name            = models.CharField(max_length=45)
+
+    class Meta:
+        db_table    = 'allergies'
+
+class ProductInformation(models.Model):
+    sale_unit       =  models.CharField(max_length=45)
+    weight_g        =  models.DecimalField(max_digits=3, decimal_places=2)
+    delivery_type   =  models.CharField(max_length=45)
+    packing_type    =  models.CharField(max_length=45)
+    instruction     =  models.TextField(null=True)
+    product         =  models.OneToOneField('Product' ,on_delete = models.CASCADE)
+    comment         =  models.CharField(max_length=100,null=True)
+    allergy         =  models.ManyToManyField('Allergy', through = 'AllergyProduct')
+
+    class Meta:
+        db_table = 'productinformations'
+
+class AllegyProduct(models.Model):
+    allergy             = models.ForeignKey('Allergy', on_delete = models.CASCADE)
+    productinformation  = models.ForeignKey('ProductInformation', on_delete = models.CASCADE)
+
+    class Meta:
+        db_table = 'allegyproducts'
```

Comment:
```suggestion
        db_table = 'allegy_products'
```
컨벤션상 이어지는 단어는 언더스코어로 구분해주세요.

Reply:
다른 테이블 이름들도 연결 단어는 변경해주세요.

Reply:
수정 완료 했습니다.